setup.py

```diff
@@ -1,10 +1,27 @@
 #!/usr/bin/env python
 from setuptools import find_packages, setup
+from setuptools.command.build_ext import build_ext
 
 long_description = (
     open("README.rst").read() + '\n\n' + open("HISTORY.rst").read()
 )
 
+class BuildClibBeforeExt(build_ext):
```

Comment:
Looks like you have a newstyle/oldstyle class problem here that means this doesn't work on Python 2.7.